config/samples/operator_v1_imageset.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: operator.tigera.io/v1
+kind: ImageSet
+metadata:
+  name: calico-v3.4.0
+spec:
+  images:
+  - image: "calico/node"
+    digest: "sha256:0deadbeef1234567890"
```

Comment:
They'll change with each release though. Its probably better to have someone use these dummy ones and fail rather than deploy with some really old images (I'm assuming they won't get updated).
WDYT?